Pipeliner/PipelinerService.swift

```diff
@@ -8,26 +8,36 @@
 import Foundation
 
 class PipelinerService {
+
     internal let dateService: DateService
-    private let resolver = ServiceResolver()
+    private let api: PipelineAPI = APIClient()
     
     init() {
         self.dateService = DateService()
     }
```

Comment:
Mby later we could instantiate all in init() or all in property definition :)

Reply:
yeah absolutely, just wanted to do this all step by step 😉